go.mod

```diff
@@ -5,7 +5,7 @@ require (
 	contrib.go.opencensus.io/exporter/ocagent v0.4.9 // indirect
 	github.com/Azure/go-autorest v11.5.2+incompatible // indirect
 	github.com/appscode/jsonpatch v0.0.0-20190108182946-7c0e3b262f30 // indirect
-	github.com/coreos/prometheus-operator v0.26.0 // indirect
+	github.com/coreos/prometheus-operator v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
Are you sure about the version?

Reply:
nope, didn't change it directly. I will try to revert and see if it works

Reply:
works, bumped to latest